src/algebra/big_operators/finprod.lean

```diff
@@ -240,13 +249,14 @@ by { classical, rw [finprod_def, dif_pos hf] }
   ∏ᶠ i : α, f i = ∏ i, f i :=
 finprod_eq_prod_of_mul_support_to_finset_subset _ (finite.of_fintype _) $ finset.subset_univ _
 
-@[to_additive] lemma finprod_mem_eq_prod_of_mem_iff (f : α → M) {s : set α} {t : finset α}
-  (h : ∀ {x}, f x ≠ 1 → (x ∈ s ↔ x ∈ t)) :
-  ∏ᶠ i ∈ s, f i = ∏ i in t, f i :=
+@[to_additive] lemma finprod_cond_eq_prod_of_cond_iff (f : α → M) {p : α → Prop} {t : finset α}
```

Comment:
```suggestion
@[to_additive] lemma finprod_cond_eq_prod_of_mem_iff (f : α → M) {p : α → Prop} {t : finset α}
```

Reply:
* `finprod_cond_eq_prod_of_mem_iff`: there is no `∈` in the LHS of `iff` in `h`.

* `finprod_cond_eq_finset_prod_iff`: this is not `∏ᶠ i (hi : p i), f i = ∏ i in t, f i ↔ ...`

Reply:
Oh, I read that as `(finprod_cond_eq_prod_of_cond)_iff` not `(finprod_cond_eq_prod)_of_(cond_iff)`. You're right, my suggestions aren't great.